.github/workflows/validate_maps_with_xsd11-validator.yml

```diff
@@ -0,0 +1,24 @@
+name: Validate map files with xsd11-validator
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  validate_map_files_with_xsd-validator:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Validate all maps
+        uses: ./.github/actions/xsd11-validator
+        with:
+          FILES: "./logical_scenarios/concrete_scenarios/road_networks/alks_road_straight.xodr \
+          ./logical_scenarios/concrete_scenarios/road_networks/alks_road_left_radius_250m.xodr \
+          ./logical_scenarios/concrete_scenarios/road_networks/alks_road_right_radius_250m.xodr \
+          ./logical_scenarios/concrete_scenarios/road_networks/alks_road_left_radius_1000m.xodr \
+          ./logical_scenarios/concrete_scenarios/road_networks/alks_road_right_radius_1000m.xodr \
+          ./logical_scenarios/concrete_scenarios/road_networks/alks_road_different_curvatures.xodr"
+          SCHEMA: ./.github/workflows/OpenDRIVE_Core.xsd
```

Comment:
Die anderen Schemas werden nicht angewendet?

Reply:
The OpenDRIVE_core.xsd refers to the other schema internally. So they are also used.